tests/apps/resiliencyapp/go.mod

```diff
@@ -12,7 +12,7 @@ require (
 
 require (
 	github.com/google/uuid v1.6.0 // indirect
-	go.opentelemetry.io/otel v1.26.0 // indirect
+	go.opentelemetry.io/otel v1.27.0 // indirect
```

Comment:
Changes to go.mod files seem irrelevant to this PR

Reply:
if you look in the go.mod in dapr it has that version for otel, but it must have been missed for the ones in this PR. I ran `make modtidy-all`